spec.html

```diff
@@ -34798,6 +34798,22 @@ <h1>%TypedArray%.prototype.set ( _array_ [ , _offset_ ] )</h1>
           </emu-alg>
         </emu-clause>
 
+        <emu-clause id="sec-is-compatible-typed-array-type" aoid="IsCompatibleTypedArrayType">
+          <h1>IsCompatibleTypedArrayType ( _type_ , _otherType_ )</h1>
```

Comment:
what does "is compatible" mean? why would Uint8ClampedArray and Float32Array be "compatible"?

Reply:
I agree this name is not quite good. I'm changing this to `IsCopyableTypedArrayType` and use Types instead of the constructor name.